src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { Table } from '../Table';
+import { getPeople } from '../../api';
+
+export const PeoplePage: React.FC = () => {
```

Comment:
let's do come structure refactoring

create 2 folders:
1. `pages` : put here components like `PeoplePage`, `HomePage`, `NotFoundPage`
2. `components`: put all other componets which we use for these pages